src/app/build/build.component.html

```diff
@@ -14,201 +16,118 @@ <h1>
             <em>{{ build.id }}</em>
           </h1>
 
-          <div class="build-status success" ng-class="{
-            'success': build.worker.status == 'Succeeded',
-            'failed': build.worker.status == 'Failed',
-            'active': build.worker.status == 'Pending'}">
-            <div class="success" ng-show="build.worker.status == 'Succeeded'">
-              <i class="icon ion-md-checkmark-circle"></i> {{ build.worker.status }}</div>
-            <!-- Pending API hookup:
-            <div class="failed ng-hide" ng-show="build.worker.status == 'Failed'"><i class="icon ion-md-close-circle"></i> Failed</div>
-            <div class="active ng-hide" ng-show="build.worker.status == 'Pending'"><img src="assets/images/rolling.svg"> In Progress</div>
-            -->
+          <div class="build-status">
+            <div class="unknown" *ngIf="!build.worker?.status"><i class="icon ion-md-radio-button-off"></i>  Log Unavailable</div>
+
+            <div class="pending" *ngIf="build.worker?.status == 'Pending'"><div class="spinner"></div> Initializing...</div>
+            <div class="running" *ngIf="build.worker?.status == 'Running'"><div class="spinner"></div> Build started...</div>
+
+            <div class="success" *ngIf="build.worker?.status == 'Succeeded'"><i class="icon ion-md-checkmark-circle"></i> {{ build.worker.status }}</div>
+            <div class="failed" *ngIf="build.worker?.status == 'Failed'"><i class="icon ion-md-close-circle"></i> {{ build.worker.status }}</div>
           </div>
 
-          <div class="grid-block horizontal build-metadata">
-            <div class="grid-content small-6 text-left">Started at {{ worker.start_time }} </div>
-            <div class="grid-content small-6 text-right">Finished at {{ worker.end_time }}</div>
+          <div class="row build-metadata">
+            <div class="small-6 text-left columns">Started at {{ build.worker.start_time }} </div>
+            <div class="small-6 text-right columns">Finished at {{ build.worker.end_time }}</div>
           </div>
 
-          <div class="horizontal build-chart build-chart-1" gantt="" data="data" ng-init="showChart = true" ng-hide="!showChart" ng-controller="jobsController">
+          <div class="horizontal build-chart build-chart-1">
 
             <div class="task task-0">
               <a href="">
```

Comment:
Should this point to the git repo?

Reply:
Some old code here that needs tidying. 

The build chart which depicts the timeline of jobs needs to be fixed up as a follow up task.
Ref #14 #124